constants/values.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path')
+const { snakeCase } = require('@keg-hub/jsutils')
+
+const rootDir = path.join(__dirname, '../')
+const reposPath = path.join(rootDir, 'repos')
+
+const repos = [
```

Comment:
We should do this dynamically. So it doesn't have to be updated any time the `repos` folder changes.
You could just use the `child_process.execSync` looks like this
```js
const { execSync } = require('child_process')
const rootDir = path.join(__dirname, '../')
const reposPath = path.join(rootDir, 'repos')

const repos = execSync('ls', { cwd: reposPath })
  .toString()
  .split('\n')
  .filter(item => item)

```

Reply:
Updated